docs/validator/running-a-validator.md

```diff
@@ -0,0 +1,252 @@
+---
+id: running-a-validator
+---
+
+# Running a Validator
+
+## Becoming a Gno.land validator
+
+The Gno.land blockchain is powered by the [Tendermint2](https://docs.gno.land/concepts/tendermint2) (TM2) consensus, which involves committing of new blocks and broadcasting votes by multiple validators selected via governance in [Proof of Contribution](https://docs.gno.land/concepts/proof-of-contribution) (PoC). While traditional Proof of Stake (PoS) blockchains such as the Cosmos Hub required validators to secure a delegation of staked tokens to join the validator set, no bonding of capital is involved in Gno.land. Rather, the validators on Gno.land are expected to demonstrate their technical expertise and alignment with the project by making continuous, meaningful contributions to the project. Furthermore, the voting power and the transaction fee rewards between validators are distributed evenly to achieve higher decentralization. From a technical perspective, the validator set implementation in Gno.land as its abstracted away into the `r/sys/val` realm ([work in progress](https://github.com/gnolang/gno/issues/1824)), as a form of smart-contract, for modularity, whereas existing blockchains include the validator management logic within the consensus layer.
+
+# Start a New Gno Chain and a Validator
+
+## 1. Initialize the configurations (required)
+
+```bash
+gnoland config init
+```
+
+## 2. Initialize the secrets (required)
+
+```bash
+gnoland secrets init
+```
+
+## 3. Set a moniker (optional)
+
+```bash
+gnoland config set moniker node01
+```
+
+## 4. Set the rpc connection address (required for connecting with other nodes)
+
+```bash
+gnoland config set rpc.laddr "tcp://0.0.0.0:26657"
+
+# similar behavior for cosmos validator
+# gaiad tx staking create-validator `--node string (default:tcp://localhost:26657)`
+```
+
+## 5. Set the validator private key (required)
+
+:::tip
+
+The key file path is relative by default.
+
+:::
+
+```bash
+gnoland config set priv_validator_key_file secrets/priv_validator_key.json
+```
+
+## 6. Set the validator state (required)
+
+:::tip
+
+The key file path is relative by default.
+
+:::
+
+```bash
+gnoland config set priv_validator_state_file secrets/priv_validator_state.json
+```
+
+## 7. Set the node key (required)
+
+:::tip
+
+The key file path is relative by default.
+
+:::
+
+```bash
+gnoland config set node_key_file secrets/node_key.json
+```
+
+## 8. Set the genesis file (required)
```

Comment:
This step generates the genesis, but doesn't set it (it's set in `--genesis` for `gnoland start`)

Reply:
da6ea307d83d82ee3b09ebd6fe5c740913bcf90f